CMakeLists_files.cmake

```diff
@@ -34,6 +34,7 @@ list (APPEND MAIN_SOURCE_FILES
       src/opm/common/utility/parameters/Parameter.cpp
       src/opm/common/utility/parameters/ParameterGroup.cpp
       src/opm/common/utility/parameters/ParameterTools.cpp
+      src/opm/common/utility/numeric/calc_cellvol.cpp
```

Comment:
Yet another thing... Use the same function name as file name. (Or vice versa, your choice).